columns/Utils.js

```diff
@@ -26,25 +27,21 @@ export function fileColFactory(fileVals = {}) {
             if (colProps.align === 'center') {
                 colProps.headerClass.push('xh-column-header-align-center');
                 colProps.cellClass.push('xh-align-center');
-                delete colProps.align;
             }
 
             if (colProps.align === 'right') {
                 colProps.headerClass.push('xh-column-header-align-right');
                 colProps.cellClass.push('xh-align-right');
-                delete colProps.align;
             }
 
             if (colProps.flex) {
```

Comment:
Might as well handle flex as we do fixedWidth with the `isNumber` check - if just for symmetry here.  (As discussed both flex:0 and fixedWidth:0 are weird choices, but we're not considering them to be entirely invalid.)

That's my only comment - I think the use of the lodash omit works well here.

Reply:
Done.